internal/topology/variables/clusterclass_variable_validation.go

```diff
@@ -53,7 +59,7 @@ func validateClusterClassVariableNamesUnique(clusterClassVariables []clusterv1.C
 				field.Invalid(
 					pathPrefix.Index(i).Child("name"),
 					clusterClassVariable.Name,
-					fmt.Sprintf("variable names must be unique. %v is repeated.", clusterClassVariable.Name),
+					fmt.Sprintf("variable name should be unique. Variable with name %q is defined more than once", clusterClassVariable.Name),
```

Comment:
Have we guidance on must vs should?

Reply:
I think should sounds nicer but must is more precise. I would prefer must as it's closer to reality :)

Reply:
I have switched everything to must except this one.
Fixed